cmd/upload.go

```diff
@@ -265,6 +268,13 @@ func uploadRun(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	var metadata map[string]interface{}
```

Comment:
Could you also move this up before the `release` var, to save an API request in cases where the metadata is invalid?

Reply:
okay